### DISCUSSION TITLE
This branch had an error being deployed

### DISCUSSION THREAD

**Discussion by KivancUlucan** [neuefische/questions#606] [Web]
I was just about to create a PR, when i got this error messages:

<img width="654" alt="Bildschirmfoto 2023-05-17 um 12 54 12" src="https://github.com/neuefische/questions/assets/117660396/f50a87f6-bd5b-4546-8500-1448f45538b9">

![Bildschirmfoto 2023-05-17 um 12 56 49](https://github.com/neuefische/questions/assets/117660396/a863ef2a-3bf6-4e68-bd11-8d572328edee)

**Comment by ThomasKirst** [comment]
Hi Kivanc,
the error message is quite explicit: it does not find your `summaryCard` component in `pages/index.js`.
Check the import statement:

<img width="600" alt="image" src="https://github.com/neuefische/questions/assets/4458383/e30e27db-075f-4793-9786-ed00e4ab2ef6">

The Mac does not bother, but Vercel is strict on lower-/upper-case named files.
– Thomas
